mono-api/mono-api.js

```diff
@@ -0,0 +1,74 @@
+'use strict'; 
+
+const http = require('https');
+const BASE_URL = 'api.monobank.ua';
+/*
+    this token should be passed from repository layer, it's here just for tests
+*/ 
+const TOKEN = 'uMLmvCFm6kA4N8mrflxayXMcZPqFUrxpm9q_CxBsMkaY';
+
+function createRequestBody(path, token) {
+    return {
+        hostname: BASE_URL,
+        path: path,
+        method: 'GET',
+        headers: {
+            'X-Token': token,
+            'Content-Type': 'application/json; charset=UTF-8',
+        }
+    };
+}
+
+function getData(path, token) {
+    const options = createRequestBody(path, token);
+    const req = http.request(options);
```

Comment:
Maybe, it'd be better something like this:

```
const req = createRequest(path, token);
```

Reply:
Should I move `http.request` to `createRequest` funciton?